### ISSUE TITLE
Players can ready up without a character

### ISSUE THREAD

**Issue by stuicey** [Apollo-Community/ApolloStation#805]
Causes a runtime error in gameticker.Setup() when a player selects ready in lobby without a character generated.

```
runtime error: Cannot read null.alternate_option
proc name: DivideOccupations (/datum/controller/occupations/proc/DivideOccupations)
  source file: job_controller.dm,334
  usr: null
  src: /datum/controller/occupations (/datum/controller/occupations)
  call stack:
/datum/controller/occupations (/datum/controller/occupations): DivideOccupations()
/datum/controller/gameticker (/datum/controller/gameticker): setup()
/datum/controller/gameticker (/datum/controller/gameticker): pregame()
ticker (/datum/controller/process/ticker): setup()
```

**Comment by halworsen** [comment]
Fixed in 0.3

**Comment by rjtwins** [comment]
Fixed closing.
